core/tests/test_apps.management.recovery_device.py

```diff
@@ -140,6 +140,53 @@ def test_exceptions(self):
             secret, share = process_slip39(words)
             self.assertIsNone(secret)
 
+    @mock_storage
+    def test_check_word_validity(self):
+        from trezor.messages import BackupType
+        from apps.management.recovery_device.word_validity import check, OK, NOK_IDENTIFIER_MISMATCH, NOK_ALREADY_ADDED, NOK_THRESHOLD_REACHED
+
+        storage.recovery.set_in_progress(True)
+
+        # nothing is stored -> should raise
```

Comment:
this comment is misleading.

instead how about: "We claim to know the backup type, but nothing is stored. That is an invalid state."